data/data-pipeline/data_pipeline/content/config/excel.yml

```diff
@@ -0,0 +1,255 @@
+---
```

Comment:
Hmm. Maybe we punt on this for today until a later PR, but having a separate YML for CSV and Excel feels a little cumbersome to me. Could we instead have one YAML with something like: 

```

      - score_name: GEOID10_TRACT
        label_csv: census_tract_id
        label_xlsx: Census tract ID
        format: string

Reply:
I respectfully disagree. This gives us total control of what each file has, not to mention I expect more custom controls on the Excel side (like configurable formatting per column) that I'm sure will come down the pipe as product plays more with this.